libmscore/cmd.cpp

```diff
@@ -3892,7 +3892,7 @@ void Score::cmd(const QAction* a, EditData& ed)
             { "del-empty-measures",         [this]{ cmdRemoveEmptyTrailingMeasures();                           }},
             { "add-audio",                  [this]{ addAudioTrack();                                            }},
             { "transpose-up",               [this]{ transposeSemitone(1);                                       }},
-            { "transpose-down",             [this]{ transposeSemitone(-1);                                      }},
+            { "transpose-down",             [this]{ transposeSemitone(-1);                                     }},
```

Comment:
This is not necessary...

Reply:
You can say that again.
Didn't even explicitly add that file through git add

Reply:
Thanks for keeping an eye out to make sure things are in conformity.